src/redist/targets/BundledManifests.targets

```diff
@@ -0,0 +1,64 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <ItemGroup>
+    <BundledManifests Include="Microsoft.NET.Workload.Android" Version="$(XamarinAndroidWorkloadManifestVersion)" WorkloadId="Microsoft.NET.Workload.Android" />
+    <BundledManifests Include="Microsoft.NET.Sdk.BlazorWebAssembly.AOT" Version="$(BlazorWorkloadManifestVersion)" WorkloadId="Microsoft.NET.Workload.BlazorWebAssembly" />
+  </ItemGroup>
+
+  <!-- Restore workload manifests via PackageReference -->
+  <ItemGroup>
+    <PackageReference Include="@(BundledManifests)" />
```

Comment:
We should use `PackageDownload` instead of `PackageReference` to download these.